### ISSUE TITLE
Missing validation data

### ISSUE THREAD

**Issue by uahsan3** [HsinYingLee/OPN#3]
Hello, 

I could not find "UCF_val.mat" file in your project page. This throws an error when I try to run the code. 

Please share that file. Thanks!

**Comment by HsinYingLee** [comment]
Hi,
For validation, you can use partial training data as the validation set. Or you can comment out the validation parts to avoid errors.
